libraries/app/database_api.cpp

```diff
@@ -244,6 +244,57 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
       graphene::chain::database&                                                                                                            _db;
       const application_options* _app_options = nullptr;
 
+      static market_ticker get_ticker_data(const market_ticker_object mto,
```

Comment:
1. Actually this is not a typical `getter` function, more suitable as a constructor of `market_ticker` class/struct. 
2. Please use constant references as parameters.
3. better to use `optional` for order book.